### PR TITLE
Implement modular multiplicative inverse

### PR DESCRIPTION
**Pull request by rehassachdeva**
Please review

### CODE REVIEW

**Comment by aktech** [comment]
@rehassachdeva Please fix merge conflicts.

File: pydsa/mod_inverse.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    >>> mod_inverse(3, 11)
+    4
+    >>> mod_inverse(2, 12)
+    "Doesn't exist."
+    """
+    a = a % m
+    for x in range(1, m):
+        if (a*x) % m == 1:
+            return x
+    return "Doesn't exist."
```
Comment (goelakash) [inline pydsa/mod_inverse.py]: Return type isn't consistent. Return -1 and mention this condition in return value of docstrings.

**Comment by goelakash** [comment]
@rehassachdeva Please fix conflicts.

**Comment by rehassachdeva** [comment]
@goelakash

**Pull request merged**